NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## [7.5.0] - 2016-08-03
```

Comment:
If you're adding this, I assume you're going to handle releasing a new version as soon as  you merge. Otherwise, I suggest leaving it off and then the release will happen when it happens (mostly so the date is accurate).